--- origsrc/mate-utils-1.2.0/configure.ac	2012-03-12 06:25:56.000000000 -0500
+++ src/mate-utils-1.2.0/configure.ac	2012-04-17 17:34:00.527391400 -0500
@@ -12,6 +12,7 @@ AC_CONFIG_MACRO_DIR([m4])
 
 AM_CONFIG_HEADER([config.h])
 AM_INIT_AUTOMAKE([dist-xz no-dist-gzip check-news])
+m4_ifdef([AM_SILENT_RULES],[AM_SILENT_RULES([yes])])
 
 MATE_COMMON_INIT
 
@@ -353,14 +354,10 @@ GTK_DOC_CHECK([1.10])
 dnl mate-doc-utils stuff
 MATE_DOC_INIT
 
-SHAVE_INIT([m4], [enable])
-
 AC_CONFIG_FILES([
 Makefile
 
 m4/Makefile
-m4/shave-libtool
-m4/shave
 
 po/Makefile.in
 
